exec/cnex/cnex.c

```diff
@@ -120,7 +120,7 @@ int main(int argc, char* argv[])
     /* ToDo: Remove this!  This is only for debugging. */
     /* gOptions.ExecutorDebugStats = TRUE; */
     _CrtSetDbgFlag(_CRTDBG_ALLOC_MEM_DF | _CRTDBG_LEAK_CHECK_DF);
-    _CrtSetBreakAlloc(218);
+    _CrtSetBreakAlloc(200);
```

Comment:
Can you remove this from the checked-in code? Your debug code really doesn't need to be here. Just maintain it locally.

Reply:
I don't know a good clean way to do that.  So, I'm going to create, yet another pull, but this one wil be the final separation of the code files.  It will include an exec.c, and cnex,.c will just become the main ,c file that contains int main(), and the code for the command line options, etc.  I've been meaning to do this for a while anyway.